compatibility/BUILD

```diff
@@ -52,12 +52,18 @@ config_setting(
     for sdk_version in sdk_versions
 ]
 
+# Instead of testing the full cartesian product of all SDK versions with
+# all platform (~= Sandbox/JSON API) versions, we pin the SDK version to be
+# the latest one. This gives us a reasonable feedback with regards to
+# maintaining backwards-compatibility without causing the test runs to grow
+# exponentially.
+sdk_head = "0.0.0"
+
 [
     sdk_platform_test(
         platform_version = platform_version,
-        sdk_version = sdk_version,
+        sdk_version = sdk_head,
```

Comment:
Don’t we want to test in both directions? sdk head vs all platforms and platform head vs all sdks.

Reply:
My bad, thanks.

Reply:
Push forced and fixed in https://github.com/digital-asset/daml/pull/8893/commits/599a135d35318d22e810e6bc86026621b5746056

I guess there could be a more elegant way of doing the list comprehension in a single expression, but I thought of keeping it simple. Let me know if you prefer it to be a single expression.